src/decorators/controller.ts

```diff
@@ -1,17 +1,14 @@
-import { Route } from "../types";
+import { Route, Controller } from "../types";
 
 /**
   Controller decorator is used to define a controller class.
   It should be used on a class.
   @param prefix - the prefix for all routes in the controller
 **/
 export function Controller(prefix: string) {
-  if (typeof prefix !== "string")
-    throw new Error("Controller decorator can only be used on a class");
```

Comment:
Do not remove this line insted fix error message

Reply:
![image](https://github.com/ikari-js/ikari/assets/56741781/da7c5850-44c3-4076-9781-8cdeff525986)

It is not necessary. Typescript already prevents it.